.travis.yml

```diff
@@ -12,11 +12,14 @@ services:
   - postgresql
 
 addons:
-  postgresql: "9.6"
+  postgresql: '9.4'
```

Comment:
I think someone updated it to 9.6. Do you want to downgrade it?

Reply:
No, let me use the 9.6